### PR TITLE
changes "command" to "⌘", implements #1433

### PR DESCRIPTION
**Pull request by seisfeld**
As suggested here (#1433) i quickly made this change for everyone to see how it looks.

xcode-cheat-sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@seisfeld thanks a lot!

File: share/goodie/cheat_sheets/json/xcode.json [hunk 1]
```diff
@@ -20,331 +20,331 @@
     "sections": {
         "Xcode menu command": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/xcode.json]: @seisfeld would you mind removing `comman` from all these headings? I just realized it's a little redundant. We should capitalize `Menu` as well for each.

**Comment by seisfeld** [comment]
@moollaza I made your suggested changes.

**Comment by jagtalon** [comment]
That was hella fast. :+1:

**Pull request merged**